tests/api/reports-orders.php

```diff
@@ -236,6 +241,11 @@ public function test_product_attributes_filter() {
 	 * Test filtering by custom product attribute(s).
 	 */
```

Comment:
```suggestion
	 *
	 * Skipped in WP 5.3 & 5.4 due to a bug in WordPress when validating arrays.
	 */
```

Reply:
Thank you for the suggestion @adrianduffell 👍 I will commit that.

Reply:
That's perfect, thanks!